app/views/saml_idp/idp/new.html.erb

```diff
@@ -5,6 +5,8 @@
 <%= form_tag do %>
   <%= hidden_field_tag("SAMLRequest", params[:SAMLRequest]) %>
   <%= hidden_field_tag("RelayState", params[:RelayState]) %>
+  <%= hidden_field_tag("SigAlg", params[:SigAlg]) %>
+  <%= hidden_field_tag("Signature", params[:Signature]) %>
```

Comment:
Broader question than this line - does this now mean if folks aren't using _this_ view they'll fail for requests that aren't signed? 

We'll either want to be able to disable this in options for compatibility (or an option regardless, some people might not care, though it seems like something people should care about 😄 ). Then we may want a new major version I suppose since this would break compatibility. Thoughts?